src/prefect/environments/storage/docker.py

```diff
@@ -67,10 +67,12 @@ class Docker(Storage):
             are included.
         - secrets (List[str], optional): a list of Prefect Secrets which will be used to populate `prefect.context`
             for each flow run.  Used primarily for providing authentication credentials.
-        - base_url: (str, optional): a URL of a Docker daemon to use when for
+        - base_url (str, optional): a URL of a Docker daemon to use when for
             Docker related functionality.  Defaults to DOCKER_HOST env var if not set
-        - tls_config: (Union[bool, docker.tls.TLSConfig], optional): a TLS configuration to pass to the Docker
-            client. https://docker-py.readthedocs.io/en/stable/tls.html#docker.tls.TLSConfig
+        - tls_config (Union[bool, docker.tls.TLSConfig], optional): a TLS configuration to pass to the Docker
+            client. [Documentation](https://docker-py.readthedocs.io/en/stable/tls.html#docker.tls.TLSConfig)
+        - **kwargs (Any, optional): Additional keyword arguments to pass to Docker's build step.
```

Comment:
Slight preference to have this as `build_kwargs={...}` rather than splitting in `**kwargs`. If a docker build kwarg conflicts with one of prefect's kwargs, there will be issues. It also leaves open the possibility of forwarding kwargs to other docker methods if needed.